### ISSUE TITLE
zathurarc not read on Mac

### ISSUE THREAD

**Issue by sebastinas** [pwmt/zathura#144]
*On GitLab by @deblacksin on Aug 23, 2019, 16:48*

---

hi,
I've been using Zathura with Vim on my Mac and its working great, but the only thing is that zathura doesn't seems to read zathurarc.
I have one placed on /etc/ , /.config/zathura/ , /local/share/zathura (the last is where zathura saves the commands history)
I want it to launch with specific recolor settings and window size, but nothing happens.
the viewer launches fine, and if I input the commands in the command line of zathura the colours change like they should.

here is the output of `zathura -l debug test.pdf`

```
Adirs-MBP:~ adir$ zathura -l debug /Users/adir/Google\ Drive/הרצאות/אינפי\ 2/test.pdf 
debug: ../girara/template.c:311: base_changed(): Variable 'session' not set.
debug: ../zathura/plugin.c:272: register_plugin(): plugin: filetype mapping added: com.adobe.pdf
debug: ../zathura/plugin.c:170: load_plugin(): Successfully loaded plugin from '/usr/local/Cellar/zathura/0.4.3/lib/zathura/libpdf-poppler.dylib'.
debug: ../zathura/plugin.c:173: load_plugin(): plugin pdf-poppler: version 0.2.9
debug: ../girara/session.c:915: girara_set_window_icon(): Loading window icon with name: org.pwmt.zathura
debug: ../zathura/zathura.c:366: init_database(): Using plain database backend.
debug: ../zathura/zathura.c:201: zathura_update_view_ppi(): monitor width: 285 mm, pixels: 1280, ppi: 114.08
debug: ../zathura/dbus-interface.c:139: name_lost(): Lost connection or failed to acquire 'org.pwmt.zathura.PID-58770' on session bus.
debug: ../zathura/content-type.c:97: guess_type_magic(): magic detected filetype: application/pdf
debug: ../zathura/file-monitor.c:134: zathura_filemonitor_new(): using glib file monitor
debug: ../zathura/zathura.c:1086: document_open(): starting file monitor
debug: ../zathura/zathura.c:673: setup_renderer(): starting renderer with cache size 15
debug: ../zathura/zathura.c:201: zathura_update_view_ppi(): monitor width: 285 mm, pixels: 1280, ppi: 114.08
debug: ../zathura/render.c:870: render_all(): Queuing resize for page 0 to 428 x 600 (612.28 x 858.90).
debug: ../zathura/callbacks.c:235: cb_widget_screen_changed(): called
debug: ../zathura/zathura.c:1204: document_open(): Failed to load bookmarks.
debug: ../zathura/jumplist.c:160: zathura_jumplist_load(): No jumplist for this file in the database yet
debug: ../zathura/render.c:918: page_cache_is_cached(): Page 1 is a cache miss
debug: ../zathura/render.c:1018: zathura_renderer_page_cache_add(): Page 1 is cached at cache index 0
debug: ../zathura/render.c:844: render_job(): Rendering page 1 ...
debug: ../zathura/render.c:520: emit_completed_signal(): Emitting signal for page 1
```

**Comment by sebastinas** [comment]
I don't know to which folder XDG_CONFIG_HOME expands to, but I suspect it will contain /Users/adir in some way or another. In any case, the latest girara version from the develop branch will print the config files zathura tries to load. Could you please update to that version and let us know the debug output?

**Comment by sebastinas** [comment]
The version printing the files zathura tries to load is now available.

**Comment by sebastinas** [comment]
*On GitLab by @deblacksin on Sep 10, 2019, 22:40*

---

I've been struggling with making "meson build" to install this since your first comment to no avail and I just gave up.. stuck on it not finding girara-gtk3 library I wish it was simpler to give up on zathura but I really can't, this is the ultimate pdf viewer!
I'll ask zegervdv the one who did the homebrew formulae for Zathura to update it to the latest version.

**Comment by sebastinas** [comment]
*On GitLab by @deblacksin on Sep 11, 2019, 14:47*

---

Some Progress!

zegervdv told me how to alter the homebrew formula and I installed 0.4.4

I than saw which folders zathura is searching for zathurarc, and made sure the file is there. zathura said "failed to read file.." so I checked it and apparently there was .txt suffix to it. I removed it and now it works. I can't tell you how happy I am. thank you so much!
